.mega-linter.yml

```diff
@@ -2,6 +2,8 @@
 DISABLE_LINTERS:
   # Has some problems reading referenced policy files. I created a separate workflow for TfLint as it creates valuable output.
   - TERRAFORM_TFLINT
+  # Seems very, very slow compared to other linters.
+  - TERRAFORM_KICS
```

Comment:
issue: This is not related to this PR. I have seen bad execution times of the KICS linter as well. The output is valuable, but we have to optimize the execution time. Idea is to run it in parallel with the Megalinter. This saves up to 3 minutes (the time it needs to pull the Megalinter image).

To be done in a separate PR.